CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 10.0.6 (unreleased)
+## 10.0.7 (unreleased)
```

Comment:
We shoudn't anticipate the next release here until you are ready to make it. See https://keepachangelog.com/en/1.1.0/ 


```suggestion
## [Unreleased]
```